package.json

```diff
@@ -26,7 +26,7 @@
     "npm-license": "0.3.3",
     "shelljs": "0.7.7",
     "shelljs-nodecli": "0.1.1",
-    "typescript": "~2.3.2"
+    "typescript": "~2.4 || ~2.3.2"
```

Comment:
Thanks, as per my comments on the issue I feel this should still just be one version for now. We can potentially look to change this philosophy before `ts-2.4` is merged into master